addons/product/models/product_product.py

```diff
@@ -18,6 +18,7 @@ class ProductProduct(models.Model):
     _inherits = {'product.template': 'product_tmpl_id'}
     _inherit = ['mail.thread', 'mail.activity.mixin']
     _order = 'priority desc, default_code, name, id'
+    _check_company_domain = models.check_company_domain_parent_of
```

Comment:
Isn't this supposed to be on the product template to match the record rule? (`product.product_comp_rule`)

Reply:
Hi @william-andre , indeed adding it to product.template makes sense :+1: